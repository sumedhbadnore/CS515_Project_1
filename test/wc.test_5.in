Testing special characters!@#$%^&*()_+
New line \n and tab \t characters.

Hopefully this should work.