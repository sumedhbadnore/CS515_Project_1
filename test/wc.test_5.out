 4 14 102 test/wc.test_5.in